nservicebus/azure/azure-servicebus-transport.md

```diff
@@ -12,33 +12,33 @@ redirects:
  - nservicebus/using-azure-servicebus-as-transport-in-nservicebus
 related:
  - samples/azure/azure-service-bus
+reviewed: 2016-03-07
 ---
 
-In some environments such as very large cloud networks or hybrid network scenarios, it is not possible or recommended to rely heavily on the DTC, and thus on MSMQ, to ensure transactional behavior and retry in case of failures. A good alternative to MSMQ in this case is to use Azure Service Bus instead.
+In some environments it is not possible or recommended to rely heavily on distributed transactions to ensure reliability and consistency. Therefore in environments such as very large cloud networks or hybrid networks using MSMQ is not the best idea. In those scenarios a good alternative is Azure Service Bus.
```

Comment:
> a good alternative is Azure Service Bus.

Should be "a good alternative are Azure transports (Azure Storage Queues or Azure Service Bus).